mikrotik/resource_bgp_instance_test.go

```diff
@@ -21,7 +21,7 @@ func TestAccMikrotikBgpInstance_create(t *testing.T) {
 
 	resourceName := "mikrotik_bgp_instance.bar"
 
-	resource.ParallelTest(t, resource.TestCase{
```

Comment:
parallel tests were the reason of failure for newer `7.x` versions.
It could be that behavior of RouterOS changed since `7.1.beta...` which we used in CI previously.
I think `ParallelTest` should be used only when it is clear that there would be no interference in those tests.

Reply:
Sounds good. The build is really fast so it's not necessary, but wanted to understand the reason.